server/api/lib/mavat/files/index.js

```diff
@@ -0,0 +1,64 @@
+const { File } = require('../../../model');
```

Comment:
file model is not exported at `api/model/index.js`. either do that or change this require